src/components/Inputs/EmailInput.tsx

```diff
@@ -25,54 +25,67 @@ export interface EmailInputProps {
   readonly disabled?: boolean;
   readonly required?: boolean;
   readonly onChange?: React.ChangeEventHandler<HTMLInputElement>;
+  readonly ref?: React.ForwardedRef<HTMLInputElement>;
 }
 
-export const EmailInput: React.FC<EmailInputProps> = ({
-  label = 'email label',
-  labelPosition = 'left',
-  gapSize,
-  fontSize,
-  fontColor,
-  placeholder = 'Enter email address',
-  placeholderColor,
-  boxShadowColor,
-  borderColor,
-  focusBorderColor,
-  focusBorderWidth,
-  width,
-  height,
-  minEmailLength = 1,
-  maxEmailLength = 254,
-  multiple,
-  pattern,
-  onChange,
-  required,
-  disabled,
-}) => (
-  <LabelPasswordInputContainer labelPosition={labelPosition} gapSize={gapSize}>
-    <label>{label}</label>
-    <StyledPasswordInput
-      fontSize={fontSize}
-      fontColor={fontColor}
-      boxShadowColor={boxShadowColor}
-      borderColor={borderColor}
-      focusBorderColor={focusBorderColor}
-      focusBorderWidth={focusBorderWidth}
-      width={width}
-      height={height}
-      placeholder={placeholder}
-      placeholderColor={placeholderColor}
-      type="email"
-      minLength={minEmailLength}
-      maxLength={maxEmailLength}
-      multiple={multiple}
-      pattern={pattern}
-      onChange={onChange}
-      required={required}
-      disabled={disabled}
-      data-testid="email-input"
-    />
-  </LabelPasswordInputContainer>
+export const EmailInput: React.FC<EmailInputProps> = forwardRef(
```

Comment:
Could we please use arrow functions for all components?

Reply:
+1

Reply:
Updated!  It was interesting to find out that arrow functions don't add screen names by default.  They have to be added to make our linter happy.